src/odemis/acq/feature.py

```diff
@@ -179,6 +180,11 @@ def __init__(self, name: str,
         self.path: str = None  # TODO:support path creation here, rather than on milling data save
         self.reference_image: model.DataArray = None
 
+        # Save the stream used for SuperZ workflow
+        self.superz_stream_name: str = None
```

Comment:
in future if you expect some `superz` related attr, its better to create a Enum now itself and store it as `superz_data` dict.